### PR TITLE
Set up repo for contribution.

### PR DESCRIPTION
**Pull request by devlyn37**
This PR:

- Adds contribution doc and MIT License
- Adds prettier, formats code accordingly

closes https://github.com/devlyn37/Safety-Goggles/issues/22

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Eventually, there will be testing requirements, but not yet.
+
+## Any contributions you make will be under the MIT Software License
+
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project.
+
+## Bugs and Enhancements are handled with [Github issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues)
+
+If you'd like to report a bug or request a new feature/improvement [create an issue](https://github.com/devlyn37/Safety-Goggles/issues), labeling each as either a bug or an enhancement respectively.
+
```
Comment (Josiassejod1) [inline CONTRIBUTING.md:27]: Maybe add a section on your roadmap and what you plan to add in the future?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Contributing to Safety Goggles
+
+Thanks for thinking of contributing!
+
+## We use Github
+
+Github is where we host code, track issues, and manage feature requests. Any additions or fixes to Safety Goggles are managed through pull requests.
+
+## Branching Strategies, deployment, and contributing.
```
Comment (Josiassejod1) [inline CONTRIBUTING.md:9]: strategies

File: README.md [hunk 3]
```diff
@@ -14,8 +14,7 @@ This website distinguishes between these fake mints and real ones, labeling them
 
 ![Screen Shot 2021-11-30 at 2 58 41 PM](https://user-images.githubusercontent.com/17352012/144141661-6c069a51-cd77-4a70-995c-1dde716656db.png)
 
-
-with this, in combination with filtering, the goal of the site is to be a tool to help discover and share alpha and hopefully prevent some people from being tricked into fomo buying something.
+With this, in combination with filtering, the goal of the site is to be a tool to help discover and share alpha and hopefully prevent some people from being tricked.
```
Comment (Josiassejod1) [inline README.md:17]: I don't even know what "share alpha" means would be good if you provided more clarity

File: components/AddressContainer.tsx [hunk 4]
```diff
@@ -48,19 +48,12 @@ export const AddressContainer: FC<{
     }, 2 * 1000);
   };
 
-  const icon = showCheck ? (
-    <FiCheckSquare className={styles.icon} />
-  ) : (
-    <FiCopy className={styles.icon} />
-  );
+  const icon = showCheck ? <FiCheckSquare className={styles.icon} /> : <FiCopy className={styles.icon} />;
```
Comment (Josiassejod1) [inline components/AddressContainer.tsx:51]: I kind of like how you had it spaced before, this feels a bit weird now with the closing tage for FiCheckSquare trailing

File: components/CopyLinkBtn.tsx [hunk 5]
```diff
@@ -18,11 +18,7 @@ export const CopyLinkBtn: FC<{}> = () => {
     }, 2 * 1000);
   };
 
-  const icon = showCheck ? (
-    <FiCheckSquare className={styles.icon} />
-  ) : (
-    <FiCopy className={styles.icon} />
-  );
+  const icon = showCheck ? <FiCheckSquare className={styles.icon} /> : <FiCopy className={styles.icon} />;
```
Comment (Josiassejod1) [inline components/CopyLinkBtn.tsx:21]: same comment as above

File: components/Event.tsx [hunk 6]
```diff
@@ -117,11 +102,7 @@ export const Event: FC<{ event: NFTEvent }> = ({ event }) => {
   };
 
   const otherAddress =
```
Comment (Josiassejod1) [inline components/Event.tsx:104]: I feel like you might want to use cases for this to be a bit clearer

File: components/Event.tsx [hunk 7]
```diff
@@ -136,54 +117,33 @@ export const Event: FC<{ event: NFTEvent }> = ({ event }) => {
...
           {(event.action === "Sold" || event.action === "Bought") && (
             <h3 className={`${styles.priceDesc} ${styles.titleItem}`}>
               {"for"}
               <div className={styles.priceContainer}>
-                <img
-                  src="/ethereum_icon.svg"
-                  className={styles.ethIcon}
-                  alt="Ethereum Icon"
-                />
+                <img src="/ethereum_icon.svg" className={styles.ethIcon} alt="Ethereum Icon" />
```
Comment (Josiassejod1) [inline components/Event.tsx:138]: might want to take the svgs and create a const file and import them for better readability

const ETHER_SVG = '/etherium_icon.svg';

```js
<img src={ETHER_SVG} />
```

File: components/Event.tsx [hunk 8]
```diff
@@ -136,54 +117,33 @@ export const Event: FC<{ event: NFTEvent }> = ({ event }) => {
...
                 {`${formatEthFromWei(event.price)}`}
```
Comment (Josiassejod1) [inline components/Event.tsx:139]: dope

File: components/Event.tsx [hunk 9]
```diff
@@ -192,23 +152,10 @@ export const Event: FC<{ event: NFTEvent }> = ({ event }) => {
               className={styles.linkIconContainer}
               style={{ marginRight: "7px" }}
             >
-              <img
-                src={"/etherscan-logo-circle.svg"}
-                className={styles.linkIcon}
-                alt="Etherscan Logo"
-              ></img>
+              <img src={"/etherscan-logo-circle.svg"} className={styles.linkIcon} alt="Etherscan Logo"></img>
```
Comment (Josiassejod1) [inline components/Event.tsx:155]: same comment as above

**Review by Josiassejod1 — APPROVED** [review]
Got some minor changes to make but this looks good

File: components/Event.tsx (hunk 7 above)
Comment (devlyn37) [reply components/Event.tsx]: I agree, not within the scope of this PR though

File: CONTRIBUTING.md (hunk 1 above)
Comment (devlyn37) [reply CONTRIBUTING.md:27]: Yeah good idea

File: CONTRIBUTING.md (hunk 2 above)
Comment (devlyn37) [reply CONTRIBUTING.md]: good catch

File: README.md (hunk 3 above)
Comment (devlyn37) [reply README.md:17]: totally

**Comment by devlyn37** [comment]
@Josiassejod1 Thanks for the feedback!

**Pull request merged**